src/main/java/com/coveo/spillway/storage/LimitUsageStorage.java

```diff
@@ -120,6 +120,28 @@
   Map<LimitKey, Integer> debugCurrentLimitCounters();
 
   /**
+   * @param resource The resource for which you want to get the current limit counts
+   * @return A Map of all the enforced limits for a resource and their current count
+   *
+   */
+  Map<LimitKey, Integer> getCurrentLimitCounters(String resource);
```

Comment:
But how do you suggest using the map that is returned? If you want to look up a specific limit, you'll need to build a `LimitKey` with a particular time bucket...